extensions/convert-px-to-vw-vh/src/menu-bar.tsx

```diff
@@ -0,0 +1,44 @@
+import { Clipboard, environment, MenuBarExtra, openCommandPreferences, showHUD } from "@raycast/api";
+import { useHistory } from "./history";
+
+export default function Command() {
+  const { history, remove, clear } = useHistory();
+
+  return (
+    <MenuBarExtra icon="../assets/logo.svg">
```

Comment:
You need to provide an image both in [light and dark](https://developers.raycast.com/api-reference/user-interface/icons-and-images#example-2), otherwise it's not possible to see

<img width="159" alt="image" src="https://github.com/raycast/extensions/assets/38033107/20f1bc4f-b9c5-49a0-b7bf-2513ade16b37">